src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeLexer.kt

```diff
@@ -31,5 +31,5 @@ class SnakemakeLexer : PythonIndentingLexer() {
         val KEYWORDS_2_TEXT = KEYWORDS.map { it.value to it.key }.toMap()
     }
 
-    override fun getTokenType() = KEYWORDS[tokenText] ?: super.getTokenType()
+    // override fun getTokenType() = KEYWORDS[tokenText] ?: super.getTokenType()
```

Comment:
This commit is completely not related to issue #16 and pull request. Let's do not mix different complicated changes in one pull request